src/components/Map/Map.tsx

```diff
@@ -0,0 +1,63 @@
+import { FC, useState } from 'react';
+import styles from './Map.module.scss';
+import { MapView } from '@aws-amplify/ui-react-geo';
+import '@aws-amplify/ui-react-geo/styles.css';
+import { NavigationControl, FullscreenControl, ScaleControl, Marker, Popup } from 'react-map-gl';
+
+interface MapProps {
+  markers: { coordinates: number[]; name: string; colour: string }[];
+}
+
+interface PopupInfo {
+  latitude: number;
+  longitude: number;
+  name: string;
+}
+
+const Map: FC<MapProps> = ({ markers }) => {
+  const [popupInfo, setPopupInfo] = useState<PopupInfo | null>(null);
+
+  return (
+    <div className={styles.container}>
+      <MapView
+        initialViewState={{ latitude: 53.708427, longitude: -1.361425, zoom: 10 }}
+        mapStyle={'VectorHereExplore'}
+        style={{ borderRadius: '20px', width: 'inherit', height: 'inherit' }}
```

Comment:
Does it need inline styles or can you add a classname and do in scss file?

Reply:
Doesn't seem to have a classname prop :(

Reply:
Boo